trieTools/zeroBalanceSystemAccountChecker/fileHandler.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/ElrondNetwork/elrond-tools-go/trieTools/zeroBalanceSystemAccountChecker/common"
+)
+
+type osFileHandler struct {
+}
+
+func newOSFileHandler() *osFileHandler {
+	return &osFileHandler{}
+}
+
+func (fh *osFileHandler) Open(name string) (io.Reader, error) {
+	return os.Open(name)
+}
+
+func (fh *osFileHandler) ReadAll(r io.Reader) ([]byte, error) {
+	return ioutil.ReadAll(r)
+}
+
+func (fh *osFileHandler) Getwd() (dir string, err error) {
+	return os.Getwd()
+}
+
+func (fh *osFileHandler) ReadDir(dirname string) ([]common.FileInfo, error) {
+	files, err := ioutil.ReadDir(dirname)
```

Comment:
why not return directly? Why do we need the copy on L36-L39?

Reply:
Because `ioutil.ReadDir` returns `[]fs.FileInfo` which contains extra information, which I do not need e.g.: 

``` go
type FileInfo interface {
	Name() string       // base name of the file
	Size() int64        // length in bytes for regular files; system-dependent for others
	Mode() FileMode     // file mode bits
	ModTime() time.Time // modification time
	IsDir() bool        // abbreviation for Mode().IsDir()
	Sys() interface{}   // underlying data source (can return nil)
}
```

Unfortunately, go cannot convert a slice of "parent" interfaces to a slice of "children" interfaces, si I need to copy them

Reply:
ok, can remain as it is.
In other situations I've used this interface. Helped me creating a good testing framework that circumvented the need of creating directories on a disk